### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by ChaosInTheCRD**
Since this separate library has existed, the README has been a copy of the main repository README.

While there is a lot of work to do on refactoring documentation, I think simplifying this README is a nice start. This makes it easier to understand what the repository serves as, and also removes the current problems observed with references to documentation in this README.

At a certain point we will aim to consolidate all witness related documentation into a single docs source location and expose that with a public frontend.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,302 +1,32 @@
...
-- Notary v2 Signature Validation
-- [Zarf](https://github.com/defenseunicorns/zarf) Integration
-- IronBank Attestor
-
-## Support
-
-[TestifySec](https://testifysec.com) Provides support for witness and other CI security tools.
-[Contact Us](mailto:info@testifysec.com)
+## Status
+This library is currently pre-1.0 and therefore should be considered unstable and subject to change.
```
Comment (jkjell) [inline README.md:11]: > This library is currently pre-1.0 and therefore should be considered unstable and subject to change.

Maybe to something like: 
> This library is currently pre-1.0 and therefore the API may be subject to breaking changes.

File: README.md [hunk 2]
```diff
@@ -1,302 +1,32 @@
...
+
+## Documentation
+For more detail regarding the library itself, we recommend viewing [pkg.go.dev](https://pkg.go.dev/github.com/testifysec/go-witness). For
+the documentation of the witness project, please view [the main witness repository](https://github.com/in-toto/witness/tree/main/docs).
+
+## Requirements
+In order to effectively contribute to this library, you will need:
+- A Unix-compatible Operating System
+- GNU Make
+- Go 1.19
```
Comment (jkjell) [inline README.md:29]: Side note: we should chat about minimum support Go version at some point. I think we use Go 1.21.x in most places.

**Review by jkjell — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (ChaosInTheCRD) [reply README.md:29]: yeah for sure - I was actually looking to tackle that when getting the development environment configuration figured out and defined.

File: README.md [hunk 3]
```diff
@@ -1,302 +1,32 @@
-[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/8164/badge)](https://www.bestpractices.dev/projects/8164)
+# go-witness
+A client library for [Witness](https://github.com/in-toto/witness), written in Go.
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/in-toto/go-witness.svg)](https://pkg.go.dev/github.com/in-toto/go-witness)
+[![Go Report Card](https://goreportcard.com/badge/github.com/in-toto/go-witness)](https://goreportcard.com/report/github.com/in-toto/go-witness)
+![OpenSSF Best Practices](https://www.bestpractices.dev/projects/8164/badge)](https://www.bestpractices.dev/projects/8164)
```
Comment (jkjell) [inline README.md:6]: Miss `[` at the beginning of the line.

**Review by jkjell — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ChaosInTheCRD) [reply README.md:6]: not sure how I managed that

**Review by jkjell — APPROVED** [review]

**Pull request merged**